src/WebJobs.Script/Diagnostics/Extensions/LoggerExtension.cs

```diff
@@ -121,15 +121,15 @@ internal static class LoggerExtension
                 new EventId(325, nameof(ScriptStartUpLoadedExtension)),
                 "Loaded extension '{startupExtensionName}' ({startupExtensionVersion})");
 
-        private static readonly Action<ILogger, Exception> _functionMetadataProviderReadingMetadata =
-            LoggerMessage.Define(LogLevel.Information,
+        private static readonly Action<ILogger, string, Exception> _readingFunctionMetadataFromProvider =
+            LoggerMessage.Define<string>(LogLevel.Information,
                 new EventId(326, nameof(FunctionMetadataManagerLoadingFunctionsMetadata)),
```

Comment:
Looks like a naming mismatch here in the EventId. The patter for all of these is to use the method name that uses the field. In this case should be "ReadingFunctionMetadataFromProvider"

Reply:
Fixed!